fix(dev/vite): Strip HTTP/2 pseudo headers from dev server requests (#12830)
remix-run/react-router: .changeset/sharp-sloths-swim.md

---
"@react-router/dev": patch
---

> Strip HTTP/2 pseudo headers from dev server requests - Fixes using HTTPS with the dev server

Reviewer: ```suggestion
Fix dev server when using HTTPS by stripping HTTP/2 pseudo headers from dev server requests
```
